hubspot-client-bundles/hbase-client-bundle/pom.xml

```diff
@@ -62,6 +62,26 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hbase</groupId>
+      <artifactId>hbase-compression-zstd</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>*</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>com.github.luben</groupId>
+      <artifactId>zstd-jni</artifactId>
```

Comment:
This and the exclusions above should not be necessary. Just let it come in transitively. Was there a reason for this approach?

Reply:
Without the exclusions, downstream consumers get this
```
[ERROR] Found banned dependency org.apache.hbase:hbase-server:jar:2.5-client-bundle-zstd-2.5-SNAPSHOT
[ERROR] Replace with com.hubspot.hbase:hbase-client-bundle or hbase-mapreduce-bundle.
[ERROR]    com.hubspot:hbase-tasks-core-shaded:1.0-SNAPSHOT
[ERROR]    \- com.hubspot.hbase:hbase-client-bundle:2.5-client-bundle-zstd-2.5-SNAPSHOT
[ERROR]       \- org.apache.hbase:hbase-server:2.5-client-bundle-zstd-2.5-SNAPSHOT
```

Reply:
Can you try excluding just org.apache.hbase/*? I'd rather keep exclusions to a minimum because it tends to make the resulting dependencies harder to reason about. The hbase deps in compression-zstd are all test scope so not necessary, but i think the test scope is getting lost in the shade. So exclusing just hbase should be enough.